lambda/mockData.ts

```diff
@@ -0,0 +1,20 @@
+export const mockProducts = [
```

Comment:
I recommend to define a type
```suggestion
export const mockProducts: Array<Product> = [
```